nixos/modules/services/misc/gitea.nix

```diff
@@ -263,6 +264,15 @@ in
         '';
       };
 
+      requireSigninView = mkOption {
+        type = types.bool;
```

Comment:
For enable options there's `mkEnableOption`. If you want e.g. a custom description you can do `mkEnableOption "REQUIRE_SIGNIN_VIEW for gitea" // { description = "..." }`.